test/e2e/play_kube_test.go

```diff
@@ -1466,4 +1466,20 @@ MemoryReservation: {{ .HostConfig.MemoryReservation }}`})
 		Expect(kube.ExitCode()).To(Equal(125))
 		Expect(kube.ErrorToString()).To(ContainSubstring(invalidImageName))
 	})
+
+	It("podman play kube applies log driver to containers", func() {
+		Skip("need to verify images have correct packages for journald")
```

Comment:
I skip this test because the other tests related to journald are also skipped: https://github.com/containers/podman/blob/e2b82e6245ed9459a37dc004a2b1d593d2835cb7/test/e2e/logs_test.go#L196